protobuf-maven-plugin/src/main/java/io/github/ascopes/protobufmavenplugin/mojo/AbstractGenerateMojo.java

```diff
@@ -582,6 +590,32 @@ public AbstractGenerateMojo() {
   @Parameter
   @Nullable File outputDescriptorFile;
 
+  /**
+   * Whether to attach the generated protobin descriptor as a Maven project artifact.
+   *
+   * <p>This is ignored if {@code outputDescriptorFile} is not provided.
+   */
+  @Parameter(defaultValue = DEFAULT_FALSE)
+  boolean outputDescriptorAttached;
+
+  /**
+   * Defines the Maven artifact type for the protobin descriptor when
+   * attached to the Maven project.
+   *
+   * <p>This is ignored if {@code outputDescriptorAttached} is false.</p>
```

Comment:
Could we add `@since 2.11.0` on these? 

I can bump the version on my side in the POMs (or there is a script to do it in scripts/).

Reply:
done